pom.xml

```diff
@@ -627,7 +627,15 @@
                 <configuration>
                     <rules>
                         <reactorModuleConvergence/>
-                        <requireUpperBoundDeps/>
+                        <requireUpperBoundDeps>
+                            <excludes>
```

Comment:
Required to compile. Bloomreach parent project cannot guarantee that they have the upper bound dependency.